Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +7,12 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return Lesson10.Cypher(message, shift);
         }
 
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            return Lesson10.DecryptCypher(message, shift);
```

Comment:
Please move the implementation here. It is redundant like this.

Reply:
Moved implementation to Decrypt